nomad/integration-tests-new.nomad

```diff
@@ -94,37 +94,39 @@ job "integration-tests-new" {
       }
     }
 
```

Comment:
Instead of commenting this out, I'd recommend commenting out the Pulumi nomad-job-deployment stuff:
`integration_tests_new_job_vars`, `integration_tests_new`

The reason:
This will fail. The job still accepts a `container_images` variable, but you're skipping that in the `integration_tests_new_job_vars`.